.github/workflows/codeql.yaml

```diff
@@ -29,8 +29,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  paths-filter:
```

Comment:
This doesn't work in push event, when pushing to dev branch, CodeQL must be run and it acts as a basis for those runs in pull requests

Reply:
you mean we should also run CodeQL although we also have docs change in our PR?

Reply:
> you mean we should also run CodeQL although we also have docs change in our PR?

I mean we need to always run CodeQL in dev branch.